### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by vsmart**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "clojurebridge.github.io",
   "private": true,
   "devDependencies": {
-    "js-yaml": "3.10.x",
-    "lumo-cljs": "1.7.x"
+    "js-yaml": "^3.13.1",
+    "lumo-cljs": "^1.10.1"
```
Comment (davidchambers) [inline package.json]: You could revert this change to see whether it fixes the test failure.

Comment (vsmart) [reply package.json]: upgrading the underlying ubuntu resolved the failure

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,4 @@
 dist: trusty
-sudo: false
```
Comment (davidchambers) [inline .travis.yml]: Why is this change necessary, out of interest?

Comment (vsmart) [reply .travis.yml]: That `sudo: false` key notation has been deprecated. From [Travis docs](https://docs.travis-ci.com/user/reference/trusty/):

> Container-based infrastructure has been fully deprecated. Please remove any `sudo: false` keys in your .travis.yml file to use the fully-virtualized Linux infrastructure.
